File: zxing-sample/src/main/java/me/dm7/barcodescanner/zxing/sample/FullScannerFragmentActivity.java
package me.dm7.barcodescanner.zxing.sample;

import android.os.Bundle;

import me.dm7.barcodescanner.zxing.sample.databinding.ActivityFullScannerFragmentBinding;

public class FullScannerFragmentActivity extends BaseScannerActivity {
    @Override
    public void onCreate(Bundle state) {
        super.onCreate(state);
        ActivityFullScannerFragmentBinding binding = ActivityFullScannerFragmentBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        setupToolbar(binding.toolbar);
    }
}

File: zxing-sample/src/main/java/me/dm7/barcodescanner/zxing/sample/FullScannerFragmentActivity.kt
package me.dm7.barcodescanner.zxing.sample

import android.os.Bundle
import me.dm7.barcodescanner.zxing.sample.databinding.ActivityFullScannerFragmentBinding

class FullScannerFragmentActivity : BaseScannerActivity() {

    override fun onCreate(state: Bundle?) {
        super.onCreate(state)

        val binding = ActivityFullScannerFragmentBinding.inflate(layoutInflater)
        setContentView(binding.root)

        setupToolbar(binding.toolbar)
    }
}
